include/cantera/oneD/Flow1D.h

```diff
@@ -55,20 +55,30 @@ class Flow1D : public Domain1D
     //!     to evaluate all thermodynamic, kinetic, and transport properties.
     //! @param nsp  Number of species.
     //! @param points  Initial number of grid points
+    //! @deprecated To be removed after %Cantera 3.2. Use constructor using Solution
+    //!     instead.
     Flow1D(ThermoPhase* ph = 0, size_t nsp = 1, size_t points = 1);
 
     //! Delegating constructor
+    //! @deprecated To be removed after %Cantera 3.2. Use constructor using Solution
+    //!     instead.
     Flow1D(shared_ptr<ThermoPhase> th, size_t nsp = 1, size_t points = 1);
 
     //! Create a new flow domain.
-    //! @param sol  Solution object used to evaluate all thermodynamic, kinetic, and
+    //! @param phase  Solution object used to evaluate all thermodynamic, kinetic, and
     //!     transport properties
     //! @param id  name of flow domain
     //! @param points  initial number of grid points
-    Flow1D(shared_ptr<Solution> sol, const string& id="", size_t points=1);
+    Flow1D(shared_ptr<Solution> phase, const string& id="", size_t points=1);
 
     ~Flow1D();
 
+private:
+    //! Initialize arrays.
+    //! @todo Consolidate after removal of legacy constructors.
+    void _init(ThermoPhase* ph, size_t nsp, size_t points);
```

Comment:
Specifying "after Cantera 3.2" would make it more clear when this todo item can be completed.